tests/lax_scipy_sparse_test.py

```diff
@@ -134,6 +203,29 @@ def test_cg_as_solve(self, shape, dtype):
         lambda x, y: lax_cg(posify(x), y),
         (a, b), order=2, rtol=1e-2)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name":
+          "_shape={}".format(jtu.format_shape_dtype_string(shape, dtype)),
+       "shape": shape, "dtype": dtype}
+      for shape in [(2, 2)]
+      for dtype in float_types + complex_types))
+  def test_bicgstab_as_solve(self, shape, dtype):
+    rng = jtu.rand_default(self.rng())
+    a = poisson(shape, dtype)  # TODO(sunilkpai): random doesn't work (bicg numerics), so using poisson instead...
+    # a = rng(shape, dtype)
```

Comment:
This is at least a little surprising. I wouldn't expect random 2x2 matrices to be a problem -- that suggests that maybe there's a bug in the algorithm?

Reply:
Yeah, I am surprised as well. I think having breakdown checks is perhaps important. If `scipy` returns `-10` or `-11` that means rho = 0 or omega = 0 respectively and maybe that is resulting in some issues. I will look into this; I do recall small matrices actually trips up even `scipy`'s `bicgstab`.